.circleci/config.yml

```diff
@@ -50,6 +51,11 @@ jobs:
             git submodule update --init
             SPHINXOPTS=-q make -e html
 
+      - run:
+          name: run doctests on documentation
+          command: |
+            . venv/bin/activate
+            python tools/refguide_check.py --rst
```

Comment:
I think this should be before the "build devdocs" step, otherwise it wastes cycles checking the documentation generated by `autodoc` from docstrings. Those are checked in a different refguide-check run on Azure, so no need to double check.

Reply:
check